docs/sbft-formal-model/replica.i.dfy

```diff
@@ -42,59 +42,80 @@ module Replica {
     map[]
   }
 
+  function NothingCommitted() : CommittedClientOperations {
+    map[]
+  }
+
+  // Define your Host protocol state machine here.
+  datatype Constants = Constants(myId:HostId, clusterConfig:ClusterConfig.Constants) {
+    // host constants coupled to DistributedSystem Constants:
+    // DistributedSystem tells us our id so we can recognize inbound messages.
+    // clusterSize is in clusterConfig.
```

Comment:
Same here, the comment doesn't seem to match the code very well...

Reply:
As above I will introduce clarifications and move the comments to the intended context.